File: SoloLearn/Secret Message/SecretMessage.c
#include <stdio.h>

int main() {
    char phrase[100];
    fgets(phrase, 100, stdin);
    
    for (int i = 0; phrase[i]; i++) {
        if (phrase[i] != ' ')
            phrase[i] = 'a' + 'z' - tolower(phrase[i]);
    }

    printf("%s", phrase);
    return 0;
}

File: SoloLearn/Secret Message/SecretMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            string phrase = Console.ReadLine().ToLower();
            foreach(char c in phrase) {
                if (c == ' ')
                    Console.Write(" ");
                else 
                    Console.Write((char)('a' + 'z' - c));
            }
        }
    }
}